<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainView</class>
 <widget class="QMainWindow" name="MainView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>932</width>
    <height>687</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainView</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Ini file</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="ini_edit">
          <property name="text">
           <string>config.ini</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="label_traj_first">
          <property name="text">
           <string>Trajectory #1: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="first_traj_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="toolTip">
           <string/>
          </property>
          <property name="statusTip">
           <string/>
          </property>
          <property name="whatsThis">
           <string/>
          </property>
          <property name="accessibleDescription">
           <string/>
          </property>
          <property name="text">
           <string>/home/pisarik/Projects/UAV/DroneEmulation/build-DroneEmulation-Desktop_Qt_5_5_1_GCC_64bit-Release/trajectory1.csv
 </string>
          </property>
          <property name="placeholderText">
           <string>Trajectory #1 path to csv</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_traj_second">
          <property name="text">
           <string>Trajectory #2: </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="second_traj_edit">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>/home/pisarik/Projects/UAV/DroneEmulation/build-DroneEmulation-Desktop_Qt_5_5_1_GCC_64bit-Release/trajectory2.csv  </string>
          </property>
          <property name="placeholderText">
           <string>Trajectory #2 path to csv</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="viewpkg::GraphicsMapView" name="graphicsView">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="load_ini_btn">
        <property name="text">
         <string>Load Ini</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="load_btn">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clear_btn">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="model_settings_group">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
        </property>
        <property name="title">
         <string/>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Detector</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="detector_combo"/>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Descriptor</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="descriptor_combo"/>
         </item>
         <item>
          <widget class="QPushButton" name="calculate_btn">
           <property name="text">
            <string>Load | Calculate</string>
           </property>
           <property name="default">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="match_btn">
           <property name="text">
            <string>Match</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="is_trajectory_show_chk">
        <property name="text">
         <string>show trajectories</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="is_orientation_show_chk">
        <property name="text">
         <string>show orientations</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="is_direction_show_chk">
        <property name="text">
         <string>show directions</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="is_key_point_show_chk">
        <property name="text">
         <string>show key points</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="is_map_show_check">
        <property name="text">
         <string>show map</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="is_matches_show_check">
        <property name="text">
         <string>show matches</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
        </property>
        <property name="title">
         <string>Shift trajectory #2</string>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>X</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="trj2_shift_x_spin">
               <property name="suffix">
                <string/>
               </property>
               <property name="minimum">
                <number>-1000</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Y</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
               <property name="margin">
                <number>-3</number>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QSpinBox" name="trj2_shift_y_spin">
               <property name="suffix">
                <string/>
               </property>
               <property name="prefix">
                <string/>
               </property>
               <property name="minimum">
                <number>-1000</number>
               </property>
               <property name="maximum">
                <number>1000</number>
               </property>
               <property name="singleStep">
                <number>5</number>
               </property>
               <property name="value">
                <number>100</number>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QPushButton" name="trj2_shift_btn">
           <property name="text">
            <string>Shift</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>932</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Open"/>
    <addaction name="separator"/>
    <addaction name="action_Exit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Exit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>viewpkg::GraphicsMapView</class>
   <extends>QGraphicsView</extends>
   <header>view/graphics_map_view.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>action_Exit</sender>
   <signal>triggered()</signal>
   <receiver>MainView</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>310</x>
     <y>231</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
